modules/lfs/transferadapter_test.go

```diff
@@ -24,9 +24,12 @@ func TestBasicTransferAdapterName(t *testing.T) {
 
 func TestBasicTransferAdapter(t *testing.T) {
 	p := Pointer{Oid: "b5a2c96250612366ea272ffac6d9744aaf4b45aacd96aa7cfcb931ee3b558259", Size: 5}
+	downloadTestFlag := false
```

Comment:
`download := false`

Reply:
ok

Reply:
wait, I forgot that it should check since the accept header is still add now